src/stabilizer_state.jl

```diff
@@ -26,11 +26,12 @@ Generates a state of n qubits in the +1 Z eigenstate.
 """
 function zero_state(n::Int)
     state = StabilizerState(n)
+    state.simulator.set_num_qubits(n)
     for i in 0:n-1
         state.simulator.z(i)
     end
     state.is_updated = false
-    update_tableau(state)
+    #update_tableau(state)
```

Comment:
[question] Why this has been commented out? Shouldn't it be deleted?

Reply:
I'll delete it (forgot to delete after checking that things still worked correctly)